static/js/services/db-sync.js

```diff
@@ -120,12 +100,8 @@ var _ = require('underscore'),
         replicateDoneCallback(null, result);
       };
 
-      var startContinuousReplication = function(params) {
-        replicate('from', {
-          filter: 'erlang_filters/doc_by_place',
```

Comment:
Should you also delete the filter? Both this one and the JS one? Or do you think we should keep it?

_(deleting the erlang filter also gives us the opportunity to delete the stuff I did to extract and upload extra ddocs, which may be a good thing unless we intend to use that code down the line)_